lib/logger.rb

```diff
@@ -577,15 +597,11 @@ class LogDeviceMutex
       include MonitorMixin
     end
 
-    def initialize(log = nil, opt = {})
+    def initialize(log, opt = {})
```

Comment:
Oh, I thought I reverted this already. I will revert. This change is not needed. 

Actually, I investigated nowhere is calling this **internal** LogDevice initializer with no argument. And, this must be wrong because we get `open` error if we call LogDevice.new with no argument. I will or may fix in another commit.